code/__HELPERS/trait_helpers.dm

```diff
@@ -152,6 +152,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_IGNOREDAMAGESLOWDOWN "ignoredamageslowdown"
 #define TRAIT_GOTTAGOFAST		"gottagofast"
 #define TRAIT_GOTTAGONOTSOFAST	"gottagonotsofast"
+#define TRAIT_GOTTAGOSLOW		"gottagoslow"
```

Comment:
I'm generally not a fan of how these traits are called without explanation. Can you add a comment describing what this does?